src/main/resources/db/changelog/changes/03-update-user-shipping-address.yaml

```diff
@@ -0,0 +1,10 @@
+databaseChangeLog:
+  -  changeSet:
+       id:  modify-shipping-address-type
+       author:  Mykhailo Kornukh
+       changes:
+         -  modifyDataType:
```

Comment:
don't add new changeset
in such case you need to implement rollback that is out of scope in this project
just change value in create-users-table changeset